src/include/OpenImageIO/imagecache.h

```diff
@@ -724,42 +724,29 @@ class OIIO_API ImageCache {
                         int miplevel, ustring dataname, TypeDesc datatype,
                         void* data);
 
-    /// Copy the ImageSpec associated with the named image (the first
-    /// subimage & miplevel by default, or as set by `subimage` and
-    /// `miplevel`).
+    /// Copy the ImageSpec associated with the named image file.
```

Comment:
I would like to suggest elaborating a bit more here:

```suggestion
    /// Copy the ImageSpec that describes the named image file.
    ///
    /// Note that the spec returned describes the file as it exists in the
    /// file, at the base (highest-resolution) MIP level of that subimage.
    /// Certain aspects of the in-cache representation may differ from the
    /// file (due to ImageCache implementation strategy or options like
    /// `"forcefloat"` or `"autotile"`). If you really need to know the
    /// in-cache data type, tile size, or how the resolution or tiling changes
    /// on a particular MIP level, you should use `get_cache_dimensions()`.
```

Reply:
Done in [this commit](https://github.com/AcademySoftwareFoundation/OpenImageIO/pull/4442/commits/f1c97b62efc06a2707810cd0cd2ae13cd7df3153)